File: PRAK604-2210817120013-Fathiah_Nuraisyah_Radam.py
kata1 = input()
kata2 = input()
a = 0
b = 0
if (len(kata1) != len(kata2)):
      print("Panjang kalimat berbeda, pesan palsu")
else:
      for i in range (len(kata1)):
             if (kata1[i] == kata2[i]):
              print("*", end =" ")
              a = a + 1
             else:
              print("#", end=" ")
              b = b + 1
      print()
      print("* = ", a)
      print("# = ", b)
      if (a >= b):
            print("Pesan Asli")
      else:
            print("Pesan Palsu")

File: PRAK604-2210817120013-Fathiah_Nuraisyah_Radam.c
#include <stdio.h>
#include <string.h>
int main(){
      char word1[50], word2[50];
      scanf(" %[^\n]%*c", &word1);
      scanf(" %[^\n]%*c", &word2);
      int kata1, kata2;
      kata1 = strlen(word1);
      kata2 = strlen(word2);
      if (kata1 != kata2){
            printf("Panjang kalimat berbeda, pesan palsu");
            return 0;
      }
      int i, a = 0, b = 0;
      for (i = 0; i < kata1; i++){
            if (word1[i] == word2[i]){
                  printf("*");
                  a++;
            }
            else {
                  printf("#");
                  b++;
            }
      }
      printf("\n");
      printf("* = %d\n", a);
      printf("# = %d\n", b);
      if (a >= b){
            printf("Pesan Asli\n");
      }
      else {
            printf("Pesan Palsu");
      }
}